src/js/utils/SetupUtil.js

```diff
@@ -28,6 +28,8 @@ const SetupUtil = {
         preparedData[key] = SetupUtil.getArrayFromHostsString(newFormData[key]);
       } else if (key === 'ssh_port') {
         preparedData[key] = parseInt(newFormData[key]);
+      } else if (key === 'nested_cluster_options') {
```

Comment:
I think the word `nested` is what's throwing me off.
`cluster_configuration`
`cluster_configuration_options`